app/javascript/packs/leafletMap.js

```diff
@@ -296,7 +290,7 @@ function recalculateTooltipVisibility() {
     floors[currentFloor].labels.eachLayer((layer) => layer.closeTooltip());
   }
 
-  if (zoomLevel <= 17) {
+  if (zoomLevel <= 18) {
```

Comment:
:thinking:

Reply:
Why do you change this feature here? It's not related to the Issue!

Reply:
Christian wanted to change this, because it looked better on desktop (albeit not on mobile). It will be reverted :sweat_smile: